neonvm/apis/neonvm/v1/virtualmachinemigration_webhook.go

```diff
@@ -42,22 +45,27 @@ var _ webhook.Validator = &VirtualMachineMigration{}
 //
 // The controller wraps this logic so it can inject extra control in the webhook.
 func (r *VirtualMachineMigration) ValidateCreate() (admission.Warnings, error) {
-	// TODO: implement creation validation webhook (?)
+	if len(r.Spec.VmName) == 0 {
+		return nil, fmt.Errorf("vmName is required")
+	}
```

Comment:
Huh, won't this already be required by the CRD? (if not: can we have it handled there? IIRC there's tons of validation options, I would assume "non-empty string" is one of them :sweat_smile:)

Reply:
You are right, I did it with `kubebuilder:validation:MinLength` in the last commit.

However, it is trickier to test.